brave/brave-core
# Displays default text with correct title for an empty tweet

Resolves https://github.com/brave/brave-browser/issues/4583

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:
- enable rewards
- go to empty tweet https://twitter.com/bravelaurenwags/status/1131664248856354816
- do inline tip for that tip
- make sure that you see this
![image](https://user-images.githubusercontent.com/9574457/58429577-a274c980-80a6-11e9-9c2e-ded9a35c55f3.png)


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
